corehq/apps/settings/templates/settings/change_my_password.html

```diff
@@ -1,6 +1,18 @@
 {% extends 'settings/bootstrap3/base_template.html' %}
 {% load i18n %}
 {% load crispy_forms_tags %}
+{% load hq_shared_tags %}
+{% block js-inline  %}{{ block.super }}
+    <script src="{% static 'zxcvbn/dist/zxcvbn.js' %}"></script>
+    <script src="{% static 'registration/js/password.js' %}"></script>
+    <script type="text/javascript">
+        PASSWORD_WEAK = "{% trans "Your password is too weak!" %}";
```

Comment:
it may be worth adding a hint like "Try adding numbers or symbols." just thinking about some of the self start users who might run up against this error and not be familiar on how to make a strong password